pandas/tests/series/test_combine_concat.py

```diff
@@ -10,7 +10,7 @@
 import pandas as pd
 from pandas import DataFrame, DatetimeIndex, Series, compat, date_range
 import pandas.util.testing as tm
-from pandas.util.testing import assert_series_equal
+from pandas.util.testing import assert_series_equal, assert_frame_equal
 
 
 class TestSeriesCombine():
```

Comment:
Since you here, have this test class subclass `object`.

Reply:
In fact, we could make this whole test file `pytest`-idiomatic with functions instead of a test class, but something to consider later for another time. 🙂